certified-connectors/Business Assist/apiDefinition.swagger.json

```diff
@@ -0,0 +1,688 @@
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/forecast": {},
+    "/api/beta/forecast/{Forecast job Id}": {
+      "get": {
+        "responses": {
+          "202": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "OperationId": {
+                  "type": "string",
+                  "description": "Forecast operation Id"
+                },
+                "Name": {
+                  "type": "string",
+                  "description": "Forecast name"
+                },
+                "Status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Forecast status"
+                },
+                "Diagnostics": {
+                  "type": "string",
+                  "description": "Diagnostics message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get forecast status",
+        "description": "Get forecast job status by job id.",
+        "operationId": "GetForecastStatus",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "Forecast job Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "a6fe7af6-ba42-4676-9a4c-901d0bd44d19",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/api/beta/forecast/{Forecast job Id}/download": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "Date": {
+                    "type": "string",
+                    "description": "Date"
+                  },
+                  "Forecast": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "Forecast"
+                  },
+                  "Low": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "Low",
+                    "title": ""
+                  },
+                  "High": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "High"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get forecast result",
+        "description": "Get forecast job result by using job id.",
+        "operationId": "GetForecastResult",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "Forecast job Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "a6fe7af6-ba42-4676-9a4c-901d0bd44d19",
```

Comment:
@sjagaleti please change the value of the description. Secret look alike values are not allowed.
Please change this in all the parameters descriptions